openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -161,11 +162,10 @@ export class Carousel {
         $.ajax({url: url, type: 'GET'})
             .then((results) => {
                 this.removeLoadingSlide();
-                const data = JSON.parse(results)
-                const cards = data.partials || []
+                const cards = results.partials || []
                 cards.forEach(card => this.slick.addSlide(card))
 
-                if (!cards.length || cards.length < loadMore.limit) {
+                if (!cards.length) {
```

Comment:
Ah this unfixes the issue Lisa was talking about 😞

Reply:
#7552 Fixed by #10652